cpp/arcticdb/storage/python_bindings.cpp

```diff
@@ -153,8 +153,12 @@ void register_bindings(py::module &m) {
         ;
 
     py::register_exception<DuplicateKeyException>(storage, "DuplicateKeyException");
-    py::register_exception<NoDataFoundException>(storage, "NoDataFoundException");
```

Comment:
I'll have to handle this in my internal -> external bridge. Why is this being removed, and what is its replacement?